checkov/terraform/checks/resource/aws/CloudWatchLogGroupRetentionYear.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class CloudWatchLogGroupRetentionYear(BaseResourceCheck):
+    def __init__(self) -> None:
+        """
+        NIST.800-53.r5 AU-10, NIST.800-53.r5 AU-11, NIST.800-53.r5 AU-6(3), NIST.800-53.r5 AU-6(4),
+        NIST.800-53.r5 CA-7, NIST.800-53.r5 SI-12
+        CloudWatch log groups should be retained for at least 1 year
+        """
+        name = "Ensure that CloudWatch Log Group specifies retention days"
```

Comment:
Could the description be updated to reflect this new requirement?

It is misleading otherwise...